src/ssl.c

```diff
@@ -7373,6 +7382,13 @@ int wolfSSL_i2d_PUBKEY(const WOLFSSL_EVP_PKEY *key, unsigned char **der)
     return wolfSSL_i2d_PublicKey(key, der);
 }
 
+int wolfSSL_i2d_X509_PUBKEY(WOLFSSL_X509_PUBKEY* x509_PubKey, unsigned char** der)
```

Comment:
Put this in x509.c with other WOLFSSL_X509_PUBKEY APIs.

Reply:
:heavy_check_mark: